configure.ac

```diff
@@ -0,0 +1,35 @@
+AC_PREREQ(2.63)
+AC_INIT([coding-shell-extensions],[0.0])
+
+AC_CONFIG_MACRO_DIR([m4])
+AC_CONFIG_AUX_DIR([config])
+
+AM_INIT_AUTOMAKE([1.10 dist-bzip2 no-dist-gzip foreign tar-ustar])
+
+m4_ifdef([AM_SILENT_RULES],[AM_SILENT_RULES([yes])])
+
+GETTEXT_PACKAGE=coding-shell-extensions
+AC_SUBST(GETTEXT_PACKAGE)
+AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE")
+
+PKG_PROG_PKG_CONFIG([0.22])
+
+GLIB_GSETTINGS
+
+shell_major=`echo "$PACKAGE_VERSION" | cut -d'.' -f1`
+shell_minor=`echo "$PACKAGE_VERSION" | cut -d'.' -f2`
+if test "$(($shell_minor % 2))" -eq 0; then
+	SHELL_VERSION="$shell_major.$shell_minor"
+fi
+AC_SUBST([SHELL_VERSION])
```

Comment:
Ah, earlier I meant that the entire block (now these are lines 19 to 24) can be removed, not just one line.